### ISSUE TITLE
v1.3.0 breaks - React.jsx: type is invalid

### ISSUE THREAD

**Issue by mikestopcontinues** [joe-bell/next-google-fonts#102]
**Describe the bug**
Hey, since updating to v1.3.0, `<GoogleFonts/>` completely breaks the app. I changed nothing else, and rolling back to v1.2.1 fixes it. Most recent versions of all packages, node v14.

**To Reproduce**

```js
export default function App({Component, pageProps}) {
  return (
    <NumerousContextProviders>
      <GoogleFonts href="https://fonts.googleapis.com/css2?family=Bitter:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Bungee&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap" />

      <Component {...pageProps} />
    </NumerousContextProviders>
  );
}
```

**Additional context**

```js
// console.log(<GoogleFonts href={...} />)

{
  '$$typeof': Symbol(react.element),
  type: { GoogleFonts: [Function: l], default: [Function: l] },
  key: null,
  ref: null,
  props: {
    href: 'https://fonts.googleapis.com/css2?family=Bitter:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&family=Bungee&family=Rubik:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap'
  },
  _owner: null,
  _store: {}
}
```

```
Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.

Check your code at _app.js:59.
    at App (webpack-internal:///./pages/_app.js:63:5)
    at AppContainer ([REPO]/node_modules/next/dist/next-server/server/render.js:25:899)
Error: Element type is invalid: expected a string (for built-in components) or a class/function (for composite components) but got: object.
    at ReactDOMServerRenderer.render ([REPO]/node_modules/react-dom/cjs/react-dom-server.node.development.js:4053:17)
    at ReactDOMServerRenderer.read ([REPO]/node_modules/react-dom/cjs/react-dom-server.node.development.js:3690:29)
    at renderToString ([REPO]/node_modules/react-dom/cjs/react-dom-server.node.development.js:4298:27)
    at Object.renderPage ([REPO]/node_modules/next/dist/next-server/server/render.js:54:851)
    at Function.getInitialProps (webpack-internal:///./node_modules/next/dist/pages/_document.js:135:19)
    at loadGetInitialProps ([REPO]/node_modules/next/dist/next-server/lib/utils.js:5:101)
    at renderToHTML ([REPO]/node_modules/next/dist/next-server/server/render.js:54:1142)
    at async [REPO]/node_modules/next/dist/next-server/server/next-server.js:107:97
    at async [REPO]/node_modules/next/dist/next-server/server/next-server.js:100:142
    at async DevServer.renderToHTMLWithComponents ([REPO]/node_modules/next/dist/next-server/server/next-server.js:132:387)
    at async DevServer.renderToHTML ([REPO]/node_modules/next/dist/next-server/server/next-server.js:133:522)
    at async DevServer.renderToHTML ([REPO]/node_modules/next/dist/server/next-dev-server.js:34:578)
    at async DevServer.render ([REPO]/node_modules/next/dist/next-server/server/next-server.js:72:236)
    at async Object.fn ([REPO]/node_modules/next/dist/next-server/server/next-server.js:56:580)
    at async Router.execute ([REPO]/node_modules/next/dist/next-server/server/router.js:23:67)
    at async DevServer.run ([REPO]/node_modules/next/dist/next-server/server/next-server.js:66:1042)
Warning: React.jsx: type is invalid -- expected a string (for built-in components) or a class/function (for composite components) but got: object.
```

**Comment by joe-bell** [comment]
Hey @mikestopcontinues, thanks for this 🙏 

I'll revert supporting named and default exports and release as a major change instead (I think that's the issue here) – clearly supporting both isn't the right strategy going forwards

**Comment by joe-bell** [comment]
I've just released [**v2.0.0**](https://github.com/joe-bell/next-google-fonts/releases/tag/v2.0.0) which should solve your problem (default export removed in favour of named)

Let me know how you get on! I'll reopen if this still occurs

**Comment by mikestopcontinues** [comment]
Worked great. Thanks!
